onnx/defs/nn/defs.cc

```diff
@@ -1811,9 +1811,17 @@ ONNX_OPERATOR_SET_SCHEMA(
             "The ratio of random dropout, with value in [0, 1). If this input was not set, "
             "or if it was set to 0, the output would be a simple copy of the input. "
             "If it's non-zero, output will be a random dropout of the scaled input, which is typically "
-            "the case during training.",
+            "the case during training. It is an optional value, if not specified it will default to 0.5.",
             "T1",
             OpSchema::Optional)
+        .Input(
+            2,
+            "training_mode",
+            "If set to true then it indicates dropout is being used for training. It is an optional value hence unless  "
+            "specified explicitly it is false and ratio will be zero in which case it will mimic inference mode where "
```

Comment:
Change to "... unless specified explicitly, it is false. If it is false, ratio is ignored and the operation mimics inference mode ..."?